cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForStaticVarCompensators.java

```diff
@@ -123,6 +123,8 @@ private boolean setRegulatingControl(CgmesRegulatingControlForStaticVarCompensat
         }
         svc.setRegulatingTerminal(terminal);
 
+        svc.setProperty("RegulatingControl", rc.regulatingControlId);
+        svc.setProperty("targetDeadBand", String.valueOf(control.getTargetDeadBand()));
```

Comment:
Same remark as for generators: if a regulating control only regulates svc, maybe we can considerate the deadband is 0 and not keep this information from the CGMES file.

Reply:
trying to keep original data about regulating control (the conformity tests contain examples of deadband != 0)